server/routes.js

```diff
@@ -100,6 +104,7 @@ app.post('/profile/signin', (req, res, next) => {
 
 });
 
-app.get('*', (req, res, next) => {
-    res.send("Testing Route");
+app.get('/', async (req, res, next) => {
```

Comment:
i'll change this back sometime soon. Easy to test the functionality on a blank route.